components/Header/index.tsx

```diff
@@ -206,6 +208,7 @@ function Header() {
 
     return (
         <>
+            {showZilpay && <ZilPay />}
```

Comment:
Why do we need this?

Reply:
Hi, I moved the zilpay component to the header component so we can use it globally especially when Modal is shown @tralkan